src/Components/WebAssembly/WebAssemblyHttpHandler/src/Microsoft.AspNetCore.Components.WebAssembly.HttpHandler.csproj

```diff
@@ -0,0 +1,17 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <NuspecFile>$(MSBuildProjectName).nuspec</NuspecFile>
+    <TargetFramework>netstandard2.1</TargetFramework>
+    <PackageId>$(MSBuildProjectName)</PackageId>
+    <AssemblyName>System.Net.Http.WebAssemblyHttpHandler</AssemblyName>
+    <AssemblyVersion>0.2.2.0</AssemblyVersion>
+    <ProduceOnlyReferenceAssembly>true</ProduceOnlyReferenceAssembly>
+    <StrongNameKeyId>Open</StrongNameKeyId>
+    <IsProjectReferenceProvider>false</IsProjectReferenceProvider>
```

Comment:
We're shipping a ref assembly for an assembly that ships in WASM in 3.2 since they didn't have the build infrastructure ready and there were some open questions about the package's naming. These values were set to mimic the runtime assembly. Since it's a reference assembly, it's built with using IncludeAssets="Compile" which `Reference` doesn't appear to support.

Reply:
Do we expect strong name signing to change in 3.1? This is going away in 5.0, so this isn't a long term problem.

Reply:
Good point about the signing change being specific to 5.0. I'm still unsure this will work correctly and suggest some thorough validation.

Reply:
It looked fine when I checked. We also have the option of just referencing the package if we absolutely needed to. The ref assembly is very small - https://github.com/dotnet/aspnetcore/blob/blazor-wasm/src/Components/WebAssembly/WebAssemblyHttpHandler/src/Microsoft.AspNetCore.Components.WebAssembly.HttpHandler.netstandard2.1.cs and isn't really meant to change

Reply:
Why did we ever ship a ref/ assembly for something outside the shared framework? And, where's the implementation project?

This would be a very useful follow-up item because it doesn't fit w/ how the rest of this branch is built. Plus that needs to change yet-again (removing almost all `*.Manual.cs` files) soon and you won't get the benefits of the simplified results.

Reply:
Thanks for the added information. We might be able to use more consistent infrastructure to build this but I leave the priority for that to you and others.